vm-virtio/src/pmem.rs

```diff
@@ -46,13 +51,33 @@ const KILL_EVENT: DeviceEventT = 1;
 // The device should be paused.
 const PAUSE_EVENT: DeviceEventT = 2;
 
-#[derive(Copy, Clone, Debug, Default)]
-#[repr(C)]
+#[derive(Copy, Clone, Debug, Default, Deserialize)]
+#[repr(C, packed)]
```

Comment:
Ditto here, in fact can you check the structs in the other PRs too.

Reply:
Kernel does not define this one as packed.

Reply:
Ignore me I think they are all supposed to be packed but the kernel is missing this one.

Reply:
Yes they're all supposed to be packed for the configurations, that's why I added the missing ones.